rudder-lang/tools/cfjson_tester

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/python3
+
+"""
+Ncf / rudder-lang compatibility tester
+
+Usage: 
+    ncf ncf-to-json <file.cf> <new.json> [<local_config_path.conf>]
+    ncf compare-json <first.json> <new.json> [<local_config_path.conf>]
+    ncf compare-cf <first.cf> <new.cf> [<local_config_path.conf>]
+"""
+
+import sys
+import os
+import configparser
```

Comment:
this should not work on python2 https://docs.python.org/2/library/configparser.html